providers/base/units/stress/boot.pxu

```diff
@@ -57,9 +58,7 @@ flags: preserve-locale
 id: cold-boot-loop-reboot1
 category_id: stress-tests/cold-boot
 _summary: Perform cold reboot 1
-_description: This is a template that will be used to generate a stress test
-  of the system boot. Specifically this is how the device will be request a
-  reboot.
+_description: Enter sleep mode after 120s.
```

Comment:
I think the delay is configurable, so this description may be misleading.

Reply:
I changed the description, thanks!